### PR TITLE
Clen 402 2

### PR DESCRIPTION
**Pull request by kleewho**
feat: New module

### CODE REVIEW

**Comment by kleewho** [comment]
@client-engineering-bot release pubnub-entities as v0.2.0

**Comment by kleewho** [comment]
@client-engineering-bot release pubnub-entities as v0.2.0

**Comment by kleewho** [comment]
@client-engineering-bot release entities as v0.2.0

**Comment by kleewho** [comment]
@client-engineering-bot release as v7.0.5

**Comment by kleewho** [comment]
@client-engineering-bot release entities as v0.2.0

**Comment by kleewho** [comment]
@client-engineering-bot help

**Review by marcin-cebo — APPROVED** [review]

File: .pubnub.yml [hunk 1]
```diff
@@ -1,9 +1,9 @@
 name: kotlin
-version: 7.0.1
+version: 7.0.5
 schema: 1
 scm: github.com/pubnub/kotlin
 files:
-  - build/libs/pubnub-kotlin-7.0.1-all.jar
+  - build/libs/pubnub-kotlin-7.0.5-all.jar
```
[the remark this answers is not in the document]
Comment (kleewho) [reply .pubnub.yml:6]: Yes. For now it's just a testing ground. I'll open new PR or bring this one to mergeable state, but for now I wanted to verify that's everything will work as I expect it to. And it didn't that's why I create a separate smaller PR with just `products.json` here https://github.com/pubnub/kotlin/pull/29. It needs to be first on master and only after that we can release modules

**Comment by kleewho** [comment]
@client-engineering-bot release pubnub-spaces as v0.1.0

**Comment by kleewho** [comment]
@client-engineering-bot release spaces as v0.1.0

**Comment by kleewho** [comment]
@client-engineering-bot release spaces as v0.1.0

**Comment by kleewho** [comment]
@client-engineering-bot release spaces as v0.1.0

**Comment by kleewho** [comment]
@client-engineering-bot release spaces as v0.1.0

**Comment by kleewho** [comment]
I think I've tested enough. I'll be back with clean PR without so much noise in it

**Pull request closed without merging**